active/NavigationView/BottomItems/BottomMenuItems2.md

```diff
@@ -0,0 +1,194 @@
+selection model for the NavigationView; clicking on them will not raise the
+[NavigationView.SelectionChanged](https://docs.microsoft.com/uwp/api/Microsoft.UI.Xaml.Controls.NavigationView.SelectionChanged)
+event.
+
+# Examples
+
+## Add footer menu items in markup
+
+```xaml
+    <muxc:NavigationView>
+        <muxc:NavigationView.MenuItems>
+            <muxc:NavigationViewItem Icon="Play" Content="main1"/>
+            <muxc:NavigationViewItem Icon="Save" Content="main2"/>
+            <muxc:NavigationViewItem Icon="Refresh" Content="main2"/>
+            <muxc:NavigationViewItem Icon="Download" Content="main2"/>
+        </muxc:NavigationView.MenuItems>
+
+        <muxc:NavigationView.FooterMenuItems>
+            <muxc:NavigationViewItem Icon="Library" Content="bottom1" Padding="0,0,12,0"/>
+            <muxc:NavigationViewItem Icon="Mail" Content="bottom2" Padding="0,0,12,0"/>
+        </muxc:NavigationView.FooterMenuItems>
+        
+    </muxc:NavigationView>
+```
+
+
+# API Notes
+
+`public IList<object> FooterMenuItems { get; }`
+
+Gets the list of objects to be used as navigation items in the footer menu.
+
+`public object FooterMenuItemsSource { get; set; }`
+
+Sets or gets the object that represents the navigation items to be used in the footer menu.
+
+# API Details
+
+```c++
+unsealed runtimeclass NavigationView : Windows.UI.Xaml.Controls.ContentControl
+{
+    {
+        Windows.Foundation.Collections.IVector<Object> FooterMenuItems{ get; };
+        Object FooterMenuItemsSource { get; set; };
+    }
+    
+    static Windows.UI.Xaml.DependencyProperty FooterMenuItemsProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty FooterMenuItemsSourceProperty{ get; };
+}
+```
+
+# Appendix
+
+## Selection
+When users invoke a navigation item, that item will become selected and show its selection indicator. 
+At most one navigation item can be selected at a given time.
+The selection indicator will animate smoothly between items in the MenuItems list and FooterMenuItems list including the optional built-in Settings menu item.
+
+In order to achieve consistent animations and allow for only one navigation item to be selected at a time, all navigation items should be in the same selection model. To achieve this, a large collection that consists of two smaller collections (specifically FooterMenuItems and MenuItems) will be used as the source.
+
+## Accessibility
+Screen readers will announce MenuItems and FooterMenuItems as two separate lists, including position in set. 
+Unlike previous behavior, the Settings item will be announced as a part of the FooterMenuItems list - specifically, its index should be announced relative to the other items in the list. For example, if there are three elements in the FooterMenuItemsList, Settings would be announced as 3 of 3.
+
+### Keyboarding
+Given the following list, keyboarding actions should be as follows:
+- Main 1
+- Main 2
+
+<!-- -->
+
+- Bottom 1
+- Bottom 2
+- Settings
```

Comment:
Should include PaneFooter elements, e.g. a "Sync now button" or "Change account" flyout.